### ISSUE TITLE
Slider labels above track

### ISSUE THREAD

**Issue by askyel** [palantir/blueprint#3569]
#### Environment

- __Package version(s)__: blueprintjs/core - v3.15.0, blueprintjs/icons - v3.7.0
- __Browser and OS versions__: Chrome - v74.0.3729.169, macOS Mojave - v10.4

#### Question

Is it possible to use the Slider component to add corresponding labels above the track bar, like the picture below?

![image](https://user-images.githubusercontent.com/7734809/58373452-b2e94080-7efc-11e9-806b-208852d5c189.png)

**Comment by adidahiya** [comment]
No, this is currently not possible, but could be a useful addition. Open to proposals for PRs.
